langchain-core/src/runnables/base.ts

```diff
@@ -124,6 +124,15 @@ export abstract class Runnable<
 {
   protected lc_runnable = true;
 
+  name?: string;
+
+  getName(suffix?: string): string {
```

Comment:
Do we need to add to the interface?

Reply:
Yes this will be used by the schemas too, and I think it makes sense to be public

Reply:
Actually I won't add to interface